datafusion/core/src/datasource/statistics.rs

```diff
@@ -160,17 +159,6 @@ pub(crate) fn create_max_min_accs(
     (max_values, min_values)
 }
 
-fn add_row_stats(
```

Comment:
This is the same as https://github.com/apache/arrow-datafusion/blob/eef654c3b0c22b1f845b1441320b8bb718ddd605/datafusion/common/src/stats.rs#L119-L126